scenarios/kubernetes_e2e.py

```diff
@@ -476,7 +477,17 @@ def main(args):
         runner_args.append(
             '--gcp-service-account=%s' % mode.add_service_account(args.service_account))
 
-    if args.build is not None:
+    if args.use_shared_build is not None:
+        # find shared build location from GCS
+        link = args.gcs_shared + os.getenv('PULL_REFS', '') + '/'
+        if args.use_shared_build:
+            link += args.use_shared_build + '-'
+        link += 'build-location.txt'
+        url = link.replace("gs://", "https://storage.googleapis.com/")
+        build_loc = urllib2.urlopen(url).read()
+        # tell kubetest to extract from this location
+        args.kubetest_args += "--extract="+build_loc
```

Comment:
I think you'll need to add a test to assert other `--extract` does not exist in config.json, or what's the expect behavior?